src/Resources/config/actions.php

```diff
@@ -11,7 +11,7 @@
  * file that was distributed with this source code.
  */
 
-namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+namespace Sonata\AdminBundle\Config;
```

Comment:
If we don't use a `Sonata\AdminBundle` namespace phpstan report an error for every sonata-internal code used here.

Do you see anything against such change @jordisala1991 ?

Reply:
Anyway, to not change this on every sonata repo, I just ignored the error